vuepress/docs/next/docs/getting-started/ent-diag.md

```diff
@@ -24,5 +24,5 @@ Diagnostic information is critical to analysis and troubleshooting, and the Enta
 ```
 
 ### Log Files
-* Log files for the bundle build and pack commands are generated for each component inside the .entando/logs directory. For a single component, it is printed to the standard output.  Run command logs are also printed to the standard output.
+* Log files for the bundle build and pack commands are generated for each component inside the .entando/logs directory. For a single component, the log is printed to the stdout.  Run command logs are also printed to the standard output.
```

Comment:
I would probably use standard output for both. Seems odd to use stdout for on and standard output for the other.